Index: zile-2.4.6/src/term_curses.c
===================================================================
--- zile-2.4.6.orig/src/term_curses.c
+++ zile-2.4.6/src/term_curses.c
@@ -24,18 +24,22 @@
 #include <stdlib.h>
 #if defined HAVE_NCURSESW_CURSES_H
 #  include <ncursesw/curses.h>
+#  include <ncursesw/term.h>
 #elif defined HAVE_NCURSESW_H
 #  include <ncursesw.h>
+#  include <term.h>
 #elif defined HAVE_NCURSES_CURSES_H
 #  include <ncurses/curses.h>
+#  include <ncurses/term.h>
 #elif defined HAVE_NCURSES_H
 #  include <ncurses.h>
+#  include <term.h>
 #elif defined HAVE_CURSES_H
 #  include <curses.h>
+#  include <term.h>
 #else
 #  error "SysV or X/Open-compatible Curses header file required"
 #endif
-#include <term.h>
 #include "gl_array_list.h"
 
 #include "main.h"
